packages/wrangler/scripts/bundle.ts

```diff
@@ -61,15 +61,15 @@ async function buildMiniflareCLI(flags: BuildFlags = {}) {
 	await build({
 		entryPoints: ["./src/miniflare-cli/index.ts"],
 		bundle: true,
-		outfile: "./miniflare-dist/index.mjs",
+		outfile: "./miniflare-dist/index.js",
 		platform: "node",
-		format: "esm",
+		format: "cjs",
```

Comment:
Why this change? Are ES Modules not good here?

Reply:
I vote switching back to ESM, if possible